UnitsNet.WindowsRuntimeComponent/GeneratedCode/Quantities/Information.WindowsRuntimeComponent.g.cs

```diff
@@ -937,64 +937,51 @@ private decimal AsBaseNumericType(InformationUnit unit)
         /// <returns>String representation.</returns>
         public override string ToString()
         {
-            return ToString(Unit);
-        }
-
-        /// <summary>
-        ///     Get string representation of value and unit. Using current UI culture and two significant digits after radix.
-        /// </summary>
-        /// <param name="unit">Unit representation to use.</param>
-        /// <returns>String representation.</returns>
-        public string ToString(InformationUnit unit)
-        {
-            return ToString(unit, null, 2);
+            return ToString(null);
         }
 
         /// <summary>
         ///     Get string representation of value and unit. Using two significant digits after radix.
         /// </summary>
-        /// <param name="unit">Unit representation to use.</param>
         /// <returns>String representation.</returns>
         /// <param name="cultureName">Name of culture (ex: "en-US") to use for localization and number formatting. Defaults to <see cref="GlobalConfiguration.DefaultCulture" /> if null.</param>
-        public string ToString(InformationUnit unit, [CanBeNull] string cultureName)
+        public string ToString([CanBeNull] string cultureName)
         {
             var provider = cultureName;
-            return ToString(unit, provider, 2);
+            return ToString(provider, 2);
         }
 
         /// <summary>
         ///     Get string representation of value and unit.
         /// </summary>
-        /// <param name="unit">Unit representation to use.</param>
         /// <param name="significantDigitsAfterRadix">The number of significant digits after the radix point.</param>
         /// <returns>String representation.</returns>
         /// <param name="cultureName">Name of culture (ex: "en-US") to use for localization and number formatting. Defaults to <see cref="GlobalConfiguration.DefaultCulture" /> if null.</param>
-        public string ToString(InformationUnit unit, [CanBeNull] string cultureName, int significantDigitsAfterRadix)
+        public string ToString(string cultureName, int significantDigitsAfterRadix)
```

Comment:
For future reference, `significantDigitsAfterRadix` is a param I think should be baked into the format string instead - similar to `double.ToString("F2")`. We did put in a whole lot of magic though on number formatting, to use scientific notation (`1.01e5`) for values smaller/larger than a certain threshold, as well as this "digits after radix" concept. Just some bits of info to bring into the whole format string discussion later.